crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -1914,14 +1920,20 @@ impl AuthorityPerEpochStore {
         }
     }
 
-    fn get_max_execution_duration_per_commit(&self) -> Option<ExecutionTime> {
+    fn get_max_execution_duration_per_commit_as_option(&self) -> Option<ExecutionTime> {
         // The old name for this config parameter referred to "cost", but the current
         // implementation of the shared object congestion tracker uses the term
         // "execution duration" to describe the same concept.
         self.protocol_config()
             .max_accumulated_txn_cost_per_object_in_mysticeti_commit_as_option()
     }
 
+    fn get_max_congestion_limit_overshoot_per_commit(&self) -> u64 {
+        self.protocol_config()
+            .max_congestion_limit_overshoot_per_commit_as_option()
+            .unwrap_or(0)
```

Comment:
```suggestion
            .unwrap_or_default()
```
This is considered more idiomatic.